guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -44,107 +44,77 @@ This prevents the upgrade process from overwriting your DNS and DHCP configurati
-[options="nowrap"]
-----
-baseurl=file:///media/sat6/Satellite
-----
-
-. Configure the {ProjectName} Maintenance repository from the ISO file.
-
-.. Copy the ISO file's repository data file for {ProjectName} Maintenance packages:
 +
-[options="nowrap"]
-----
-# cp /media/sat6/Maintenance/media.repo /etc/yum.repos.d/satellite-maintenance.repo
-----
-
-.. Edit the `/etc/yum.repos.d/satellite-maintenance.repo` file:
+Run this command once for each of the three repositories listed above.
+.. Ensure that the repositories are synchronized on your connected server with the download policy set to `immediate`.
+.. Export each repository in a syncable format by using its repository ID:
 +
-[options="nowrap"]
-----
-# vi /etc/yum.repos.d/satellite-maintenance.repo
-----
-
-... Change the default `InstallMedia` repository name to `{Project}-Maintenance`:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-[{Project}-Maintenance]
-----
-
-.. Add the `baseurl` directive:
-+
-[options="nowrap"]
+[options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-baseurl=file:///media/sat6/Maintenance/
+$ hammer content-export complete repository \
+--id=_Repo_ID_ --format=syncable
 ----
+.. Copy the exported directories to the disconnected {ProjectServer}.
 
-ifdef::katello,satellite,orcharhino[]
-. If you use {advisorengine} in your environment, update to the latest images:
-* To update using the {Project} installation ISO image, set up containers on your {ProjectServer}:
+. On the disconnected server, stop all {Project} services:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# /media/sat6/containers/setup_containers
-----
-* Alternatively, to update by transferring the latest image from a connected system to a disconnected {ProjectServer}, follow these steps:
-... On the connected system, export the container image:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# skopeo copy containers-storage:registry.redhat.io/satellite/iop-advisor-engine-rhel9:{ProjectVersion} \
-oci-archive:/tmp/iop-advisor-engine.tar
+[options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-... Transfer the `iop-advisor-engine.tar` file to the disconnected {ProjectServer}.
-... On the disconnected {ProjectServer}, import the container image:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
+# {foreman-maintain} service stop
 ----
-# skopeo copy oci-archive:/tmp/iop-advisor-engine.tar \
-containers-storage:registry.redhat.io/satellite/iop-advisor-engine-rhel9:{ProjectVersion}
+. On the disconnected server, enable access to the exported content by configuring a repository:
+.. Identify the location of the copied exported directories from the syncable export on the connected server.
+.. Create the `/etc/yum.repos.d/upgrade.repo` file and update the `baseurl` directive to point to that location:
++
+[source, none, options="nowrap" subs="attributes"]
+----
+[{RepoRHEL9BaseOS}]
+--name=Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs) \
```

Comment:
@parthaa The double hyphens `--` at the beginning don't look like [allowed yum config syntax](https://linux.die.net/man/5/yum.conf). Has this been tested? Should I fix this?

Reply:
Fixing here> https://github.com/theforeman/foreman-documentation/pull/4388